src/components/ComponentDemo/Code/CodeBar.js

```diff
@@ -17,21 +17,24 @@ const StorybookLink = ({ framework, url }) => (
   </a>
 );
 
-const CodeBar = ({ src, code, links }) => {
+const CodeBar = ({ src, code, links, showSandboxURL }) => {
   const sandboxUrl = useCodesandbox(code);
   const storybookLinks = Object.entries(links);
 
   return (
     <div className={codeBar}>
       <div className={linkList}>
-        <a target="_blank" rel="noopener noreferrer" href={sandboxUrl}>
-          CodeSandbox <Launch16 />
-        </a>
+        {showSandboxURL && (
```

Comment:
Love the re-use of the code bar. I'm confused how `showSandboxURL` is meant to be used though. As is, this check would mean we have to add `showSandboxURL=true` to all of our deployed ComponentDemos (they're currently missing the sandbox link in build preview).

What if instead, `useCodesandbox` returned null when no code is provided. We could use `sandboxUrl &&` to determine whether to show the link or not.

```suggestion
        {sandboxUrl && (
```

Reply:
good catch